module-ballerina-openapi/Ballerina.toml

```diff
@@ -8,3 +8,6 @@ path = "../openapi-validator/build/libs/openapi-validator-@project.version@.jar"
 groupId = "ballerina"
 artifactId = "openapi"
 version = "@project.version@"
+
+[platform.java17]
+graalvmCompatible = true
```

Comment:
How did we verify that the module is graalVM compatible? Have we configured a graalVM workflow to run module test using the native executable?

Reply:
Since we only have ballerina code here, we can confirm its compatible right?

Reply:
My mistake. We do have java dependencies in the module.

Reply:
yeah we do use our core modules in here. so might need to come up with atleast a basic test suite to run on graalVM  before marking as `graalVM-compatible`.

Reply:
I'm curious about adding tests to verify GraalVM compatibility, especially since we have CLI and OpenAPI validator functionality that operates during compilation. I'll look into incorporating tests to ensure compatibility with GraalVM. 
Appreciate it if anyone has any references or insights related to GraalVM compatibility testing, please feel free to share them :)

Reply:
Looping in @TharmiganK

Reply:
This is tool right? Since we are not executing the bal tools as GraalVM native images it is not required to be GraalVM compatible

Reply:
@TharmiganK this is not the tool. We do have this `ballerina/openapi` library module to provide some annotation related to our openAPI validation support. So I think we should consider marking it as graalvm compatible (or not).

Reply:
I see
Are these annotations only relevant when we call OAS generation? 

If we do not have any runtime use cases we can mark it as GraalVM compatible to avoid the warnings when building (`bal build/test --native` or `bal pack`)

Reply:
Btw why do we have a java dependency in the `Ballerina.toml`? https://github.com/ballerina-platform/openapi-tools/blob/321ac864fe03d5328375d20fcdfc8284cec0a319/module-ballerina-openapi/Ballerina.toml#L6

Isn't this for the compiler plugin? I saw the same jar in the `CompilerPlugin.toml` : https://github.com/ballerina-platform/openapi-tools/blob/321ac864fe03d5328375d20fcdfc8284cec0a319/module-ballerina-openapi/CompilerPlugin.toml#L6

I think just removing it from the `Ballerina.toml` should work